src/components/shared/AlertBox.tsx

```diff
@@ -0,0 +1,119 @@
+import DangerousOutlinedIcon from '@mui/icons-material/DangerousOutlined';
+import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
+import ReportProblemOutlinedIcon from '@mui/icons-material/ReportProblemOutlined';
+import TaskAltOutlinedIcon from '@mui/icons-material/TaskAltOutlined';
+import {
+    Alert,
+    AlertColor,
+    AlertTitle,
+    Theme,
+    Typography,
+} from '@mui/material';
+import { alertBackground, alertTextPrimary } from 'context/Theme';
+import { ReactNode, useMemo } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { BaseComponentProps } from 'types';
+
+interface Props extends BaseComponentProps {
+    severity: AlertColor;
+    short?: boolean;
+    hideIcon?: boolean;
+    title?: string | ReactNode;
+}
+
+const SHARED_STYLING = {
+    borderRadius: 4,
+    borderTopRightRadius: 0,
+    borderBottomRightRadius: 0,
+};
+
+const HEADER_MESSAGE = {
+    warning: 'alert.warning',
+    success: 'alert.success',
+    info: 'alert.info',
+    error: 'alert.error',
+};
+
+function AlertBox({ short, severity, hideIcon, title, children }: Props) {
```

Comment:
Replacement for alerts. Easier this way than messing with the default styling and allows us to inject the new big header.